.openshift-ci/sensor_versions.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+"""
+Returns the latest patches of the last 4 major versions
```

Comment:
Either `def getLastSensorVersionsFromGitTagsCLI(num_versions):` should loose `num_versions` parameter or the description here should be modified.
Would be even better if you can restore the `main` function that simply calls and prints `getLastSensorVersionsFromGitTagsCLI` output. This way anyone will be able to run this script as a stand-alone executable and see how it works and what it produces.

Reply:
And this clears up the comment above, in that case I'll rename `sensor_versions.py` to `get_latest_release_versions.py` too